Examples/UIExplorer/UIExplorer/Info.plist

```diff
@@ -36,13 +34,23 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>NSLocationWhenInUseUsageDescription</key>
+	<string>You need to add NSLocationWhenInUseUsageDescription key in Info.plist to enable geolocation, otherwise it is going to *fail silently*!</string>
 	<key>UIViewControllerBasedStatusBarAppearance</key>
 	<false/>
+	<key>RCTWebSocketExecutor</key>
+	<dict>
+		<key>protocol</key>
+		<string>http:</string>
+		<key>hostname</key>
+		<string>localhost</string>
+		<key>port</key>
+		<integer>8081</integer>
```

Comment:
Could we just have URL here instead of parts? Seems like in the end these will get glued together anyways. IMHO it's much easier to deal with URLs.